src/classes/COFF_Affiliation_TEST.cls

```diff
@@ -121,43 +121,86 @@ public with sharing class COFF_Affiliation_TEST {
         System.assertEquals(1, affls.size());
     }
 
+    /*********************************************************************************************************
```

Comment:
I'm sorry the diff here is so bad. I believe I added one unit test, and decided to just update two other unit tests to test with more than one record being inserted/updated at one time. I originally wanted to create new unit tests for the bulk insert/update scenarios, but there are already 24 unit tests in this class...

Reply:
Yea, I'm with you on wanting to do that, but I do know that historically they have a concern with numerous individual tests...it bugs me, but I get it :/